### PR TITLE
[demo] Add Cart service exemplars doc

### PR DESCRIPTION
**Pull request by julianocosta89**
This PR adds documentation to the Exemplars added to the Cart service in the Demo.

It depends on the following issues to be merged: 
- https://github.com/open-telemetry/opentelemetry-demo/pull/1830
- https://github.com/open-telemetry/opentelemetry-demo/pull/1836

### CODE REVIEW

**Comment by julianocosta89** [comment]
`npm run fix:htmltest-config` is not changing anything 🤔

File: content/en/docs/demo/services/cart.md [hunk 1]
```diff
@@ -87,12 +87,85 @@ Action<ResourceBuilder> appResourceBuilder =
...
         .AddOtlpExporter());
 ```
 
+### Exemplars
+
+[Exemplars](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exemplars)
+are configured in the Cart service with trace-based exemplar filter, which
+enables the OpenTelemetry SDK to attach exemplars to metrics.
+
+First it is created a `CartActivitySource`, `Meter` and two `Histograms`. The
```
Comment (puckpuck) [inline content/en/docs/demo/services/cart.md:104]: Grammar

```suggestion
First it creates a `CartActivitySource`, `Meter` and two `Histograms`. The
```

**Review by puckpuck — APPROVED** [review]
One grammar nitpick, but this looks great otherwise.

**Comment by julianocosta89** [comment]
I need to update the screenshot based on @puckpuck's change in the Dashboard

**Review by chalin — CHANGES_REQUESTED** [review]
Since the image is specific to the cart service, I suggest the following:

- Create `content/en/docs/demo/services/cart/`
- Move and rename `content/en/docs/demo/services/cart.md` to `content/en/docs/demo/services/cart/index.md`
- Move and rename `content/en/docs/demo/screenshots/cart-service-exemplars.png` to `content/en/docs/demo/services/cart/exemplars.png`
- Adjust the in-page link to the image: `Cart Service Exemplars`

/cc @svrnm

**Review by svrnm — APPROVED** [review]

File: content/en/docs/demo/services/cart/index.md [hunk 2]
```diff
@@ -87,12 +87,85 @@ Action<ResourceBuilder> appResourceBuilder =
...
         .AddProcessInstrumentation()
         .AddRuntimeInstrumentation()
         .AddAspNetCoreInstrumentation()
+        .SetExemplarFilter(ExemplarFilterType.TraceBased)
         .AddOtlpExporter());
 ```
 
+### Exemplars
+
+[Exemplars](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exemplars)
```
Comment (chalin) [inline content/en/docs/demo/services/cart/index.md:100]:
```suggestion
[Exemplars](/docs/specs/otel/metrics/data-model/#exemplars)
```

**Review by chalin — APPROVED** [review]
I won't have time to do a detailed review of the prose, but overall LGTM, as does the PR structure.

**Comment by chalin** [comment]
We have all necessary approvals. Merging.

**Pull request merged**